python/llm/example/CPU/HF-Transformers-AutoModels/Model/llama2_gptq/README.md

```diff
@@ -0,0 +1,73 @@
+# Llama2
```

Comment:
1) move to `bigdl/python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/GPTQ`
2) `# GPTQ`
3) This example shows how to directly run 4-bit GPTQ models using BigDL-LLM on Intel CPU

Reply:
fixed